Source/Utils/ProfilerWrapper.H

```diff
@@ -0,0 +1,20 @@
+/* Copyright 2020 Maxence Thevenet
```

Comment:
Should this file be removed? It looks like it is not actually used.

Reply:
Indeed, thanks!